### DISCUSSION TITLE
Roadmap 2023

### DISCUSSION THREAD

**Discussion by glushchenko** [glushchenko/fsnotes#1585] [Announcements]
About the future of FSNotes 7. Usually in the summer I sit down to work on a big update for macOS, this year there won't be one as usual. And there's only one reason for that, and it's not war.

We've got a lot of technical debt piling up. The program hasn't gotten worse or more cluttered, but I would like to see more attention to detail. Details are why we love Apple tech, and FSNotes is no exception.

The program needs improvements to current functionality. Performance, bug fixes. 6.4.0 is a start, I'd also like to improve the app's loading speed, responsiveness of the text editor, etc. Close some requests.

Of course there will probably be new features, but within the scope of version 6. The main thing is attention to detail, to what is under the hood.

I would also like to say a big thank you to all the contributors. I will not list the names, so as not to forget anyone. Together we are creating the best note manager for macOS.

**Comment by gingerbeardman** [comment]
I think this is the right approach, and I will support you.

**Comment by ghost** [comment]
> Together we are creating the best note manager for macOS.

You are. Thank you.

**Comment by apastuszak** [comment]
Totally agree on this. Whatever you need to do to make your life easier to maintain the app is welcome. This is future-thinking.

🇺🇦

**Comment by dylanseeger** [comment]
I support this decision wholeheartedly and would even be in favor of removing features which add considerable complexity. Lightweight, fast, and reliable software is the best software.

**Reply by glushchenko** [reply]
Definitely yes! For me first candidate to removing is note and background color. The program looks terrible with the any settings. I don't know how many users use it, but it's completely tasteless.

**Comment by ReessKennedy** [comment]
I am already blown away by all the great features! So many things no other app I've tried can do like Git sync for multiple repos?!? Man!!!!! I just realized FSNotes can do all this after a long period of dissatisfaction with other choices. Feels like a great gift from the heavens. Thanks for all your hard work @glushchenko
